modules/sdk-unified-wallet/src/unifiedWallets.ts

```diff
@@ -95,8 +95,12 @@ export abstract class UnifiedWallets implements IUnifiedWallets {
     return newWallet;
   }
 
-  async getAllUnifiedWallets(): Promise<UnifiedWallet[]> {
-    return await this.bitgo.get(this.bitgo.url(this.urlPath, 2)).result();
+  async getUnifiedWallets(
+    paginationOptions: PaginationOptions = { prevPage: 0, limit: 5, order: 'ASC' }
```

Comment:
default limit should not be 5, too small for each page
default order should not be ascending, should be descending order by creation date

Reply:
I also was concern that too big of a default could create some issues in the front end. Any preference what it should be? (from reading the [BitGo guidelines](https://bitgoinc.atlassian.net/wiki/spaces/ENG/pages/193495049/REST+API+conventions), I didn't see suggested default)